src/api/v3/pages.py

```diff
@@ -36,10 +36,20 @@ def transform_page(page_translation):
 def pages(request, region_slug, language_code):
     region = Region.get_current_region(request)
     result = []
-    for page in region.pages.all():
+    for page in Page.objects.filter(region=region, parent=None):  # get main level
```

Comment:
```suggestion
    for page in region.pages.filter(archived=False, parent=None):  # get main level
```
archived pages should be excluded from the result.